cmd/root.go

```diff
@@ -240,15 +243,44 @@ func (r *Root) execute() {
 		Use:     "keploy",
 		Short:   "Keploy CLI",
 		Example: rootExamples,
+		Version: utils.Version,
 	}
+
 	rootCmd.CompletionOptions.DisableDefaultCmd = true
+
 	rootCmd.SetHelpTemplate(rootCustomHelpTemplate)
 
 	rootCmd.PersistentFlags().BoolVar(&debugMode, "debug", false, "Run in debug mode")
 
-	// Manually parse flags to determine debug mode early
+	rootCmd.PersistentFlags().Bool("version", false, "Fetch the latest version")
+
+	// Manually parse flags to determine debug mode and version flag early
 	debugMode = checkForDebugFlag(os.Args[1:])
-	// Now that flags are parsed, set up the l722ogger
+
+	rootCmd.SetVersionTemplate(`
+	{{with .Version}}{{printf "Keploy %s" .}}{{end}}
```

Comment:
<img width="1440" alt="Screenshot 2024-01-27 at 12 01 58 AM" src="https://github.com/keploy/keploy/assets/72094895/686ea1fb-d51d-4643-8ed1-3fee82438545">

I have tested the changes I have observed few things 

1. Once I have ran the command you can see that the indentation is lost in the version sentence and also next prompt from the terminal. Please correct the indentation in the template.
 2. Update info is given before version statement.
 3. When I was trying the command actually it took 1 or 2 secs to give out the command output. This is because of update function which is doing a  https call. We should take a call whether we should show this update thing. Because I can clearly see a lag after typing the command. This may affect the user experience a lot. Let me discuss internally too and I will let you know.
 
 If you want you can remove this update thing from this PR as these changes are present in update PR too. So that I will merge this PR and we can work regarding this update fucntion in update PR.

Reply:
I think the correct decision would be to take a step back and reflect upon the goals of this Pull Request. I agree with you that the update message should therefore then be shifted to the update command PR. Refactoring the same